pysolr.py

```diff
@@ -386,10 +386,12 @@ def _send_request(self, method, path='', body=None, headers=None, files=None):
                       url, method, log_body[:10], end_time - start_time)
 
         if int(resp.status_code) != 200:
-            error_message = self._extract_error(resp)
-            self.log.error(error_message, extra={'data': {'headers': resp.headers,
-                                                          'response': resp.content}})
-            raise SolrError(error_message)
+            error_message = "Solr responded with error (HTTP %s): %s"
```

Comment:
I'm not sure that it's worth a commit but do you think this would read better as “Solr responded with an error”?